src/app/tabs/styles/run-tab-styles.js

```diff
@@ -170,13 +175,21 @@ var css = csjs`
   .networkItem {
     margin-right: 5px;
   }
-  .clearinstance {
-    font-size: 20px;
-    cursor: pointer;
-    margin-right: 10px;
-  }
+  .clearinstance {}
   .transactionActions {
-    float: right;
+    display: flex;
+    width: 70px;
+    justify-content: space-between;
+    border: 1px solid ${styles.rightPanel.runTab.additionalText_Color};
+    padding: 5px;
+    border-radius: 3px;
+}
+
```

Comment:
... standard checking should not pass here....

Reply:
ah it is because this is inside the string

Reply:
@yann300 All good?